vehicles.ini

```
@@ -39,16 +39,17 @@ formularenault20,Formula Renault 2.0,7,7200
 formularenault35,Formula Renault 3.5,6,9000
 mx5 mx52016,Global Mazda MX-5 Cup,6,6400
 hpdarx01c,HPD ARX-01c,6,9500
-indypropm18,Indy Pro 2000 PM-18,6,8650
+indypropm18,Indy Pro PM-18,6,8650
```

Alias please revert or provide reason
```suggestion
indypropm18,Indy Pro 2000 PM-18,6,8650
```